backend/apps/sandbox/views.py

```
@@ -185,30 +243,41 @@ def export_zip(self, request, pk=None):
 
 
 class ProjectFileViewSet(viewsets.ModelViewSet):
+    """
+    ViewSet for ProjectFile model.
+    """
     serializer_class = ProjectFileSerializer
     permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)